mountpoint-s3/src/data_cache.rs

```diff
@@ -11,6 +11,7 @@ mod in_memory_data_cache;
 mod multilevel_cache;
 
 use async_trait::async_trait;
+use strum_macros::IntoStaticStr;
```

Comment:
As discussed in person, let's revert adding the dependency for now and reconsider if/when applicable to more error types.